### ISSUE TITLE
larger images make "every" fail, never repeats

### ISSUE THREAD

**Issue by pseudotensor** [gradio-app/gradio#5424]
### Describe the bug

larger images make "every" fail, never repeats

### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues

### Reproduction

Feel free to make "every=3" larger.  It has no effect, not about how long takes to render itself it seems.
```
import gradio as gr
import time

from apscheduler.schedulers.background import BackgroundScheduler

with gr.Blocks() as demo:
    text = gr.Textbox()
    image1 = gr.Image(type='filepath')

    def make_text():
        print("text", flush=True)
        return str(time.time()), "percpu.svg"

    demo.load(make_text, outputs=[text, image1], every=3)
demo.queue(concurrency_count=4, api_open=False)

demo.launch()
```

[percpu.svg.zip](https://github.com/gradio-app/gradio/files/12529808/percpu.svg.zip)

### Screenshot

![image](https://github.com/gradio-app/gradio/assets/2249614/3d4c2693-4bdf-4270-90f6-2673921b307d)

### Logs

```shell
I can't really work around it, just have to drop larger plots else all updates cannot be seen.  Uncertain exact problem, so I just have silly threshold for size of file for now.
```

### System Info

```shell
gradio==3.41.0
gradio_client==0.3.0
```

### Severity

Blocking usage of gradio

**Comment by abidlabs** [comment]
Thanks @pseudotensor I can reproduce the issue -- it's freezing up my screen a bit as well, but no errors in the console.

**Comment by freddyaboulton** [comment]
Doesn't happen when I use `gr.Video` or `gr.File` with a very large file (40 mb) so I think it may be the conversion to base64. Should be handled in 4.0.

**Comment by pseudotensor** [comment]
FYI the use case is pulling in "glances" graphs as svg into gradio.  So it's not an absurd use case :)  I convert the svg to png if svg is too large, but I'm unclear the exact cause so unsure how far it goes.

I still see hangups sometimes even after my work-arounds.

**Comment by freddyaboulton** [comment]
Uploading files as opposed to converting to base64 fixed the issue on 4.0.2!

![big_gif_every](https://github.com/gradio-app/gradio/assets/41651716/df67e7e3-9673-40d6-8c54-95c59196e30d)
